package.json

```diff
@@ -86,7 +86,7 @@
     "babel-loader": "^8.2.2",
     "codelyzer": "^6.0.0",
     "cypress": "6.1.0",
-    "eslint": "^7.6.0",
+    "eslint": "7.21.0",
```

Comment:
If the warning regarding the outdated ts version still remains with 7.21.0 I would discard these changes and do a proper upgrade of all depedencies after the redesign.

Reply:
Locally I had installed eslint 7.21.0 - got the warnings.

Upgraded to 8.0 - no warnings but an unhandled exception:
`An unhandled exception occurred: Failed to load plugin '@angular-eslint' declared in '.eslintrc.json#overrides[0]': Class extends value undefined is not a constructor or null`

Installed 7.6.0 (as in the original package.json) - got the warnings again.

So I run `git revert 53c1c84705788e0beef9b0ba0541569837e31f9e` (the commit which changed package.json and yarn.lock).
Hopefully the repo is ok now.